lib/ssz_ex.ex

```diff
@@ -106,6 +106,15 @@ defmodule LambdaEthereumConsensus.SszEx do
     chunks = value |> pack_bits(type)
     leaf_count = chunk_count(schema) |> next_pow_of_two()
     root = chunks |> merkleize_chunks_with_virtual_padding(leaf_count)
+
+    root =
```

Comment:
nit: it seems you want to create two different guards:

def hash_tree_root(value, {type, _size} = schema) when type == :bitlist
def hash_tree_root(value, {type, _size} = schema) when type == :bitvector

so that you can avoid the `if`

Reply:
I used this to avoid repeating the first 3 lines in both functions.